pg/src/main/java/org/jdbi/v3/pg/PostgresJdbiPlugin.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.pg;
+
+import org.jdbi.v3.Handle;
+import org.jdbi.v3.spi.JdbiPlugin;
+
+public class PostgresJdbiPlugin implements JdbiPlugin {
+    @Override
+    public Handle customizeHandle(Handle handle) {
+        handle.registerArgumentFactory(new SqlArrayArgumentFactory());
+        handle.registerColumnMapper(new SqlArrayMapperFactory());
```

Comment:
Something I forgot to bring up in the extensions PR: now that JdbiPlugin has a `customizeDbi` method, we should decide whether core plugins should be customizing the handle or the DBI.

Because anything registered on the handle will *always* take precedence over anything registered on the DBI, my vote would be to always register things on the DBI. That way the order of plugin registration results in a predictable order of precedence for arguments, mappers, collectors, and extensions they register.